src/fal/packages/environments/__init__.py

```diff
@@ -1,8 +1,19 @@
-from typing import List
+from typing import Optional, List
+from pathlib import Path
 
 from fal.packages.environments.base import BASE_CACHE_DIR, BaseEnvironment
+from fal.packages.environments.conda import CondaEnvironment
 from fal.packages.environments.virtual_env import VirtualPythonEnvironment
 
 
-def create_environment(*, requirements: List[str]) -> BaseEnvironment:
-    return VirtualPythonEnvironment(requirements=requirements)
+
+def create_environment(*, pip_requirements: Optional[List[str]] = None, conda_requirements: Optional[List[str]] = None) -> BaseEnvironment:
+    if conda_requirements is not None:
+        if pip_requirements:
+            raise RuntimeError("Pip based installations are not supported when using conda environments.")
+        return CondaEnvironment(conda_requirements)
+    elif pip_requirements is not None:
+        assert not conda_requirements
+        return VirtualPythonEnvironment(pip_requirements)
+    else:
+        raise RuntimeError("Either provide requirements (for pip) or conda_requirements.")
```

Comment:
What do we do if a script/model has no package requirements?

Reply:
This happens when you define an environment and that environment doesn't have any dependencies, like the following:
```yaml
environments:
  - name: not-funny
```

if you don't want to include any dependencies, you can still do it by using an empty list literal (YAML syntax)
```yaml
    # This is going to construct an empty (yet still isolated) environment.
    requirements: []
```